src/content/lodash/string/kebabCase/lodash.js

```diff
@@ -1,11 +1,51 @@
 // https://lodash.com/docs/#kebabCase
-import { kebabCase } from 'lodash'
+import { kebabCase } from 'lodash';
 
-exports.first = kebabCase('Foo Bar')
+module.exports.fooSpaceBar = kebabCase('Foo Bar');
```

Comment:
do you mind following the pattern `exports.varname`, instead of `module.exports.varname`? on the top of my mind, I'm not sure if `module.exports` gets correctly replaced in the UI, but also for consistency reasons. It should work all the same afaik

Reply:
copy/paste Error ^^